### PR TITLE
Add properties to intercom

### PR DESCRIPTION
**Pull request by krakover**

### CODE REVIEW

File: Providers/IntercomProvider.m [hunk 1]
```diff
@@ -44,6 +44,10 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
     }
 }
 
+- (void)setUserProperty:(NSString *)property toValue:(NSString *)value {
+    [Intercom updateUserWithAttributes:@{[property lowercaseString]: value}];
```
Comment (liorr) [inline Providers/IntercomProvider.m]: What happens when intercom gets an attribute it doesn't recognize?

Comment (krakover) [inline Providers/IntercomProvider.m]: they ignore it. There is a way to add custom properties (that is different from this one), but I am not sure it's required (nor I don't want to hold the entire list of options in the code)

Comment (orta) [inline Providers/IntercomProvider.m]: it might be wise to protect against nil in this method

Comment (liorr) [inline Providers/IntercomProvider.m]: +1, thanks

**Comment by liorr** [comment]
Fix and merge

**Pull request merged**